src/components/Widgets/Widgets.scss

```diff
@@ -3,7 +3,7 @@
     right: 0;
     bottom: 0;
     width: fit-content;
-    z-index: 121;
+    z-index: 150;
```

Comment:
Leave a comment regarding the z-index value here so that it will be easier to find the value we need next time.